framework-crates/objc2-virtualization/Cargo.modified.toml

```diff
@@ -0,0 +1,17 @@
+[package]
+description = "Bindings to the Virtualization framework"
+
+[dependencies]
+objc2 = { path = "../../crates/objc2", version = "0.5.1", default-features = false }
+block2 = { path = "../../crates/block2", version = "0.5.0", default-features = false, optional = true }
+objc2-app-kit = { path = "../objc2-app-kit", version = "0.2.0", default-features = false, optional = true }
+
+[features]
+default = ["std", "apple"]
+
+# Currently not possible to turn off, put here for forwards compatibility.
+std = ["alloc", "objc2/std", "block2?/std", "objc2-foundation/std"]
+alloc = ["objc2/alloc", "block2?/alloc", "objc2-foundation/alloc"]
+
+# Runtime selection. See `objc-sys` for details.
+apple = ["objc2/apple", "block2?/apple", "objc2-foundation/apple"]
```

Comment:
All of this should only be necessary when allowing GNUStep

Reply:
Removed in [039aec0](https://github.com/madsmtm/objc2/pull/613/commits/039aec0d14717fc9aaca99a1e707ed453c5a8cf2).